doc/plugins.dox

```diff
@@ -329,24 +422,30 @@ Now we need to clean up properly when the plugin is unloaded.
 To remove the menu item from the Tools menu, you can use gtk_widget_destroy().
 gtk_widget_destroy() expects a pointer to a GtkWidget object.
 
-First you should add gtk_widget_destroy() to your plugin_cleanup() function.
-The argument for gtk_widget_destroy() is the widget object you created earlier in
-plugin_init(). To be able to access this pointer in plugin_cleanup(), you need to move
-its definition from plugin_init() into the global context so its visibility will increase
+First you should add gtk_widget_destroy() to your @ref GeanyPluginFuncs::cleanup() function. The
```

Comment:
Am 16.08.2015 um 16:22 schrieb Colomban Wendling:
>
> Hum… okay, but as |gtk_widget_destroy()| won't have a link I'd find it 
> more logical to makr it code/whatever
>

We should either mark all refs to functions with @c or none, regardless 
of whether there is a link (e.g. in case we add an output format that 
doesn't have links at all [latex?]).

But since refs to functions are ubiquitous throughout that document I'm 
afraid @c would make the text harder to read overall (you can say that 
about the links already), IMO.